src/test/java/seedu/address/logic/parser/ClientNoteAddCommandParserTest.java

```diff
@@ -0,0 +1,67 @@
+package seedu.address.logic.parser;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NOTE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.ClientNoteAddCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.note.Note;
+import seedu.address.model.tag.Tag;
+
+class ClientNoteAddCommandParserTest {
```

Comment:
Same issue for this, refer to `TagNoteMapTest.java` for the comments on test method naming. For stuff that throws exception, the format should be `..._throwsXXXException()`.

Reply:
I've updated it to follow the naming standards.